sample_configs/local_configs.yaml

```diff
@@ -14,3 +14,4 @@ label: stable
 amlb_benchmark: test
 amlb_task: iris
 amlb_constraint: test
+amlb_custom_branch: https://github.com/gidler/autogluon#gidler_master_w_logging
```

Comment:
I just put this here as an example since this is a sample config. We can remove if you want.

Reply:
I got an error when doing a test run
```
ERROR: No matching distribution found for autogluon.core[all]==0.7.0b20230417
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'autogluon.tabular'
...
run_subprocess
    raise subprocess.CalledProcessError(retcode, process.args, output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/home/ubuntu/src/autogluon-bench_back/benchmark_runs/tabular/test_local_20230417T074044/automlbenchmark/frameworks/AutoGluon/setup.sh', 'gidler_master_w_logging', 'https://github.com/gidler/autogluon']' returned non-zero exit status 1.
```